packages/shopify-codemod/transforms/remove-redundant-returns-from-addeventlistener.js

```diff
@@ -0,0 +1,28 @@
+export default function removeAddEventListener(file, api) {
+  const j = api.jscodeshift;
+  return j(file.source)
+    .find(j.CallExpression, {
+      callee: {
+        type: 'MemberExpression',
+        property: {
+          name: 'addEventListener',
+        },
+      },
+    })
```

Comment:
If the `find` was more specific, the `forEach` wouldn't have to do so much logic.

```
      callee: ... same as before ...,
      arguments: {
        1: {
          type: j.ArrowFunctionExpression.name,
          body: {
            body: (lines) => j.ReturnStatement.check(lines[lines.length-1])
          }          
        }
      }
```

The pro of this is that it keeps selection / transforming in separate blocks of code.  Easy for people to comprehend.

The con of this is that it makes smarter transforms harder, but creating a separate transform for edge cases is usually the right way to go.